charms/kfp-api/src/charm.py

```diff
@@ -309,9 +310,11 @@ def _get_interfaces(self):
         try:
             interfaces = get_interfaces(self)
         except NoVersionsListed as err:
-            raise ErrorWithStatus(str(err), WaitingStatus)
+            raise ErrorWithStatus((err), WaitingStatus)
```

Comment:
should it be string or should we remove () ?

Reply:
Pushed. Let's keep it he way it was.